lightningd/onchain_control.c

```diff
@@ -46,7 +46,7 @@ static void onchaind_tell_fulfill(struct channel *channel)
 	}
 }
 
-static void handle_onchain_init_reply(struct channel *channel, const u8 *msg)
+static void handle_onchain_init_reply(struct channel *channel)
 {
```

Comment:
Correct fix here is to actually fromwire_onchain_init_reply() the message, I think.  That's more uniform and more future proof.